packages/create-astro/src/data/festive.ts

```diff
@@ -0,0 +1,66 @@
+import { random } from '@astrojs/cli-kit/utils';
+
+export default function getFestiveHouston(fancy?: boolean) {
+	const date = new Date();
+	if (date.getMonth() === 11) {
+		// Christmas season
+		return {
+			clothes: {
+				hat: random(['🎁', '🎄', '🌲']),
+				tie: '🧣'
```

Comment:
Technically these should all respect the `fancy` option, which is a good sign that we don't actually need to do that logic in this function. We can do it in the `actions/context.ts` file after we call `getFestiveHouston`/`getSeasonalHouston`.